controllers/app.controllers.js

```diff
@@ -40,3 +40,15 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticlesId = (req, res, next) => {
+  const id = req.params;
+  const articleId = id.article_id
+  return Promise.all([selectArticlesByID(articleId), selectCommentsByArticleId(articleId)])
```

Comment:
There is no need to send the article back on the response. We do want to know if that article exists though as this will determine the reason for there being no comments see the comments on missing tests.